.github/workflows/feature_5_legacy_docs.yml

```diff
@@ -63,5 +63,5 @@ jobs:
     - name: (deploy) Upload website to S3
       run: aws s3 sync --acl=public-read --delete ./_site $(cat S3_BUCKET)/docs
       env:
-        AWS_ACCESS_KEY_ID: ${{ secrets.STAGING_AWS_ACCESS_KEY_ID }}
-        AWS_SECRET_ACCESS_KEY: ${{ secrets.STAGING_AWS_SECRET_ACCESS_KEY }}
+        AWS_ACCESS_KEY_ID: ${{ secrets.STAGING_LEGACY_AWS_ACCESS_KEY_ID }}
```

Comment:
We should form a standardised way of documenting the link between secrets and their corresponding LastPass entries.

Reply:
Yeah we should as it would have helped a lot here, for now though that's a problem for another day. Would be nice if GitHub provided a way to add notes on the secret tbh.